MonacoEditorTestApp/MainPage.xaml.cs

```diff
@@ -360,7 +360,7 @@ private async void ButtonSetMarker_Click(object sender, RoutedEventArgs e)
             }
             else
             {
-                //Editor.Markers.Clear();
+                Editor.Markers.Clear();
                 await Editor.SetModelMarkersAsync("CodeEditor", Array.Empty<IMarkerData>());
```

Comment:
Wouldn't we need only one and not the other? Trying to remember why the original line was commented out now.